arviz/plots/distcomparisonplot.py

```diff
@@ -0,0 +1,203 @@
+        backend_kwargs = {}
+
+    datasets = []
+    groups = []
+    for group in all_groups:
+        try:
+            datasets.append(getattr(data, group))
+            groups.append(group)
+        except:  # pylint: disable=bare-except
+            pass
+
+    if var_names is None:
+        var_names = list(datasets[0].data_vars)
+
+    if isinstance(var_names, str):
+        var_names = [var_names]
+
+    if isinstance(var_names[0], str):
+        var_names = [var_names for _ in datasets]
+
+    var_names = [_var_names(vars, dataset) for vars, dataset in zip(var_names, datasets)]
+
+    if transform is not None:
+        datasets = [transform(dataset) for dataset in datasets]
+
+    datasets = get_coords(datasets, coords)
+    len_plots = rcParams["plot.max_subplots"] // (len(groups) + 1)
+    len_plots = len_plots if len_plots else 1
+    dc_plotters = [
+        list(xarray_var_iter(data, var_names=var, combined=True))[:len_plots]
+        for data, var in zip(datasets, var_names)
+    ]
+
+    nvars = len(dc_plotters[0])
+    ngroups = len(groups)
+
+    (figsize, _, _, _, linewidth, _) = _scale_fig_size(figsize, textsize, 2 * nvars, ngroups)
+
+    posterior_kwargs.setdefault("plot_kwargs", dict())
+    posterior_kwargs["plot_kwargs"]["color"] = vectorized_to_hex(
+        posterior_kwargs["plot_kwargs"].get("color", "C0")
+    )
+    posterior_kwargs["plot_kwargs"].setdefault("linewidth", linewidth)
+    posterior_kwargs.setdefault("hist_kwargs", dict())
+    posterior_kwargs["hist_kwargs"].setdefault("alpha", 0.5)
+
+    prior_kwargs.setdefault("plot_kwargs", dict())
+    prior_kwargs["plot_kwargs"]["color"] = vectorized_to_hex(
+        prior_kwargs["plot_kwargs"].get("color", "C1")
+    )
+    prior_kwargs["plot_kwargs"].setdefault("linewidth", linewidth)
+    prior_kwargs.setdefault("hist_kwargs", dict())
+    prior_kwargs["hist_kwargs"].setdefault("alpha", 0.5)
+
+    observed_kwargs.setdefault("plot_kwargs", dict())
+    observed_kwargs["plot_kwargs"]["color"] = vectorized_to_hex(
+        observed_kwargs["plot_kwargs"].get("color", "C2")
+    )
+    observed_kwargs["plot_kwargs"].setdefault("linewidth", linewidth)
+    observed_kwargs.setdefault("hist_kwargs", dict())
+    observed_kwargs["hist_kwargs"].setdefault("alpha", 0.5)
+
+    distcomparisonplot_kwargs = dict(
+        ax=ax,
+        nvars=nvars,
+        ngroups=ngroups,
+        figsize=figsize,
+        dc_plotters=dc_plotters,
+        legend=legend,
+        groups=groups,
+        prior_kwargs=prior_kwargs,
+        posterior_kwargs=posterior_kwargs,
+        observed_kwargs=observed_kwargs,
+        backend_kwargs=backend_kwargs,
+        show=show,
+    )
+
+    if backend is None:
+        backend = rcParams["plot.backend"]
+    backend = backend.lower()
+    if backend == "bokeh":
+        warnings.warn(
+            "The bokeh backend is still under development. Using matplotlib bakend.", UserWarning
+        )
+    backend = "matplotlib"
```

Comment:
This should be moved to the backends, the high level plots shouldnt refer to backend at all. Also would prefer if plot doesnt load at all if bokeh is used and instead just raises warning